frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -926,6 +926,18 @@ export function ContainerInfo(props: ContainerInfoProps) {
                 )}
               </>
             ))}
+            {container.ports?.length! > 5 &&
+              portsToShow &&
+              container.ports &&
+              portsToShow.length < container.ports.length && (
+                <Grid item xs={12}>
+                  <Box display="flex" mt={2}>
+                    <Button onClick={() => setMaxPortsToShow(prev => prev * 2)}>
+                      {t('translation|Show More')}
```

Comment:
I tested it for a 1000 ports exposed, I am not sure if that's something which is likely in a real world scenario.

Reply:
I have added a show all commit fixup today to check how it felt. I think in a real use-case scenario, users are unlikely to want to discover ports gradually by doing the show more. Normally they want to see whether a certain port is there or to see all of them. I understand we cannot show them all by default, because it slows down the whole view if we have a lot of ports to display, but if that's a consequence of a user action (clicking "Show All") then I think it's acceptable.
Now sure why the tests are failing as I did update the snapshots.